build.gradle

```diff
@@ -14,6 +12,17 @@ buildscript {
     }
 }
 
+plugins {
+    id "io.gitlab.arturbosch.detekt" version "1.0.0-RC14"
```

Comment:
Any reason we're using this version instead of the latest version?

Reply:
For the first integration, I wanted to stick with the old RC version we are using in other projects. Once we prove that it's working as expected, I want to upgrade to the latest version, as this version is a year old. I am ticketing the upgrade to the latest version.